modules/web/src/app/wizard/step/cluster/component.ts

```diff
@@ -465,7 +469,7 @@ export class ClusterStepComponent extends StepBase implements OnInit, ControlVal
       [Controls.OPAIntegration]: this._builder.control(clusterSpec?.opaIntegration?.enabled ?? false),
       [Controls.Konnectivity]: this._builder.control(clusterSpec?.clusterNetwork?.konnectivityEnabled ?? true),
       [Controls.MLALogging]: this._builder.control(clusterSpec?.mla?.loggingEnabled ?? false),
-      [Controls.KubernetesDashboardEnabled]: this._builder.control(clusterSpec?.kubernetesDashboard?.enabled ?? true),
+      [Controls.KubernetesDashboardEnabled]: this._builder.control(clusterSpec?.kubernetesDashboard?.enabled ?? false),
```

Comment:
Sorry I might not be clear in my previous comment but what I meant to say was that if dashboard is enabled in admin settings then we shouldn't enable it forcefully. However, we should keep this default `true` value here.
So here are different scenarios:
- Default value should be `true` here like before.
- If dashboard is **disabled** in Admin Settings then we should set `KubernetesDashboardEnabled` control value to `false` (like you did in previous changes but add an `if` condition).
- Inside the `_loadClusterDefaults` method, we should check value of `isKubernetesDashboardEnable` first:
 ```
[Controls.KubernetesDashboardEnabled]:
            (this.isKubernetesDashboardEnable && clusterSpec?.kubernetesDashboard?.enabled) ?? this.controlValue(Controls.KubernetesDashboardEnabled)